dev-docs/bidders/orbitsoft.md

```diff
@@ -13,12 +13,12 @@ prebid_1_0_supported : true
 
 ### bid params
 
-| Name   | Scope    | Description | Example         |
-| :---   | :----    | :---------- | :------         |
-| placementId   | required | The placement ID (site channel ID)        | 142     |
-| requestUrl   | required | Url to perform search request        | "http://adserver.com/ads/show/hb"     |
-| style     | optional | Creative styles. Actual only for text ads | ... |
-| customParams | optional | Permits passing any publisher key-value pairing into the bid request     | { macro_name: "macro_value" } |
+| Name         | Scope    | Description                                                          | Example                           | Type      |
+|--------------|----------|----------------------------------------------------------------------|-----------------------------------|-----------|
+| placementId  | required | The placement ID (site channel ID)                                   | 142                               | `integer` |
+| requestUrl   | required | Url to perform search request                                        | "http://adserver.com/ads/show/hb" | `string`  |
+| style        | optional | Creative styles. Actual only for text ads                            | ...                               | `string`  |
+| customParams | optional | Permits passing any publisher key-value pairing into the bid request | { macro_name: "macro_value" }     | `object`  |
```

Comment:
Looks like this one needs `{: .table .table-bordered .table-striped }` to format the table.